app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -334,16 +378,53 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
             }
         });
 
+        addReminderBtn.setOnClickListener(view -> {
+
+            Calendar c = Calendar.getInstance();
+
+            Log.i(TAG, "onCreate: addReminderBtn clicked");
+
+            if (expiryField.getText().toString().equals("Never")) {
```

Comment:
This check will only work if an user's device is in English, please check the tempLoyaltyCard variable instead